src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -84,9 +84,10 @@ export default {
       // as default if there is any available balance. Otherwise, we return the first balance over 0
       // TODO: change to be preferrably the same token that is shown as claimed, although not so important
       if (this.balances && this.balances.length > 0) {
-        return this.balances.filter(({ denom }) => denom === this.denom)[0] &&
-          this.balances.filter(({ denom }) => denom === this.denom)[0].amount >
-            0
+        const stakingDenomBalance = this.balances.filter(
+          ({ denom }) => denom === this.denom
+        )[0]
```

Comment:
```suggestion
        const nunZeroBalances = this.balances.filter(
          ({ amount }) => amount > 0
        )
```

Reply:
I think I just applied this successfully